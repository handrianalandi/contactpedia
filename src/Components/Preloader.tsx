import React from "react";
import Skeleton from "./Skeleton";

export default function Preloader() {
  return (
    <>
      <Skeleton />
      <br />
      <Skeleton />
      <br />
      <Skeleton />
    </>
  );
}
