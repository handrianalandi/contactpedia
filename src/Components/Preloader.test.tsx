import {render} from '@testing-library/react';
import Preloader from './Preloader';
describe('Preloader', () => {
    it('should render correctly', () => {
        const {container} = render(<Preloader />);
        expect(container).toMatchSnapshot();
    }
    );
});