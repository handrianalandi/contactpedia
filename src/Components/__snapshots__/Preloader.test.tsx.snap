// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Preloader should render correctly 1`] = `
<div>
  <div
    class="css-c62n0o"
  />
  <div
    class="css-dbrq19"
  />
  <div
    class="css-7hekhs"
  />
  <br />
  <div
    class="css-c62n0o"
  />
  <div
    class="css-dbrq19"
  />
  <div
    class="css-7hekhs"
  />
  <br />
  <div
    class="css-c62n0o"
  />
  <div
    class="css-dbrq19"
  />
  <div
    class="css-7hekhs"
  />
</div>
`;
